src/test/resources/uk/gov/justice/hmpps/prison/api/resource/impl/csra_ratings.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "offenderNo": "A1183JE",
+    "classificationCode": "HI",
+    "classificationDate": "2019-01-05"
+  },
+  {
+    "offenderNo": "A1234BB"
```

Comment:
Is this right? Feels strange to return data without any csra rating data.

Reply:
i just couldn't think of a good reason to add extra code to remove them. If you think it is more consistent then I will change it.

Reply:
It probably makes sense to filter them out, If only to reduce the amount of noise that a request for a large data set could cause.